router/src/server.rs

```diff
@@ -967,7 +970,35 @@ pub async fn run(
         .layer(Extension(compute_type))
         .layer(Extension(prom_handle.clone()))
         .layer(OtelAxumLayer::default())
-        .layer(cors_layer);
+        .layer(cors_layer)
+        .layer(
+            ::tower_http::trace::TraceLayer::new_for_http()
+                .make_span_with(RequestSpan::new())
+                .on_request(|req: &::axum::http::Request<_>, _span: &::tracing::Span| {
+                    tracing::info!("Request: {} {}", req.method(), req.uri(),);
+                })
+                .on_response(
+                    |res: &::axum::http::Response<_>,
+                     latency: ::std::time::Duration,
+                     _span: &::tracing::Span| {
+                        tracing::info!(
+                            took = latency.as_secs_f32(),
+                            status_code = res.status().as_u16(),
+                            "Response: {}",
+                            res.status().as_u16(),
+                        );
+                    },
+                )
+                .on_failure(
+                    |error: ::tower_http::classify::ServerErrorsFailureClass,
+                     latency: ::std::time::Duration,
+                     _span: &::tracing::Span| {
+                        ::tracing::warn!(took = latency.as_secs_f32(), "Failure: {error:?}");
+                    },
+                ),
+        )
```

Comment:
Do we need this? We already have tracing in the server.
Can't the request id just be set by a parent span and we keep the current tracing logic?